src/VirtoCommerce.QuoteModule.Web/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Does this file really need?

Reply:
Not sure, but it's in most (or every) current v3 module